entries/fadeIn.xml

```diff
@@ -40,7 +40,7 @@ $( "#clickme" ).click(function() {
       <img class="column three" src="/resources/0042_06_34.png" alt=""/>
       <img class="column three" src="/resources/0042_06_35.png" alt=""/>
       <img class="column three" src="/resources/0042_06_36.png" alt=""/>
-      <figcaption>figure 1</figcaption>
+      <figcaption>Figure 1 - Illustration of the <code>fadeIn()</code> effect</figcaption>
```

Comment:
I think this is ok, since the group of images is composing the one figure, only the figure needs a figcaption. http://html5doctor.com/the-figure-figcaption-elements/ mentions this case for example.